HaaLimits/python/HaaLimits.py

```diff
@@ -71,138 +73,122 @@ def initializeWorkspace(self):
         self.addX(*self.XRANGE,unit='GeV',label=self.XLABEL)
         self.addMH(*self.SPLINERANGE,unit='GeV',label=self.SPLINELABEL)
 
-    def buildModel(self,region='PP',**kwargs):
+    def buildModel(self, region='PP', addUpsilon=False, setUpsilonLambda=False, sidesModeled=False, **kwargs):
         tag = kwargs.pop('tag',region)
+        if not sidesModeled:
```

Comment:
I don't like this, adding the PDFs to the workspace doesnt hurt anything.
So remove this if check.